src/curve.js

```
@@ -2,12 +2,15 @@
 'use strict';
 
 const curveJs = require('curve25519-js');
-const nodeCrypto = require('crypto');
+const { webcrypto } = require('crypto');
+const subtle = webcrypto.subtle;
+
+// DER prefixes for X25519 keys (used for WebCrypto import/export)
 // from: https://github.com/digitalbazaar/x25519-key-agreement-key-2019/blob/master/lib/crypto.js
 const PUBLIC_KEY_DER_PREFIX = Buffer.from([
```

ESLint

[error] 'PUBLIC_KEY_DER_PREFIX' is assigned a value but never used.

(no-unused-vars)